src/Microbiome.jl

```diff
@@ -32,8 +33,11 @@ import DataFrames: DataFrame
 import Clustering: Hclust, hclust
 import Base: getindex, setindex, length
 
+using Reexport
+@reexport using SpatialEcology
```

Comment:
This allows reexporting all the SpatialEcology functions - it may be overcool, and you could choose to just reexport those that you think a user of the Microbiome package would be using (notable you're not using all the spatial information)

Reply:
Fine with this.

Reply:
I wrote "overcool", I meant "overkill" :joy:

Reply:
I figured, though I'd really love it if this package is overcool 😆